test/fixtures/compute_instance/simple/versions.tf

```diff
@@ -15,5 +15,5 @@
  */
 
 terraform {
-  required_version = "~> 0.12.6"
+  required_version = ">=0.12.6"
```

Comment:
How come are tests still in 0.12.6? Is this what's in upstream?

Reply:
Yes this is what we have in the master branch. It was changed to support 0.12.6.